src/gei/gei.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.11.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="6.0.0" />
+    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
```

Comment:
isn't this needed in the ado2gh.csproj also?

Reply:
It should, at least if `ado2gh` was setting the executable bit.

Fixed in [37d14c581](https://github.com/github/gh-gei/pull/253/commits/37d14c581a1c2cbb1076efc89f48c877aa5c7a8f)